docs/ExpectAPI.md

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
i think this newline broke the generation pipeline :)

follow the steps i described to test the documentation generation and you'll see the details